docs/providers/tencent/guide/deploying.md

```diff
@@ -0,0 +1,58 @@
+# Tencent-SCF - Deploying
+
+The Serverless Framework was designed to provision your Serverless Cloud Functions, Events and infrastructure Resources safely and quickly. It does this via a couple of methods designed for different types of deployments.
+
+## Deploy All
+
+This is the main method for doing deployments with the Serverless Framework:
+
+```bash
+serverless deploy
+```
+
+Use this method when you have updated your Function, Event or Resource configuration in `serverless.yml` and you want to deploy that change (or multiple changes at the same time) to Amazon Web Services.
```

Comment:
Is it correct that we're referring "Amazon Web Services" here? Or should it be "Tencent Cloud"?

Reply:
> Is it correct that we're referring "Amazon Web Services" here? Or should it be "Tencent Cloud"?

I'm so sorry about that, it should be Tencent Cloud here.